plenum/client/client.py

```diff
@@ -27,9 +27,9 @@
 from plenum.common.signer import Signer
 from plenum.common.stacked import NodeStack
 from plenum.common.startable import Status, LedgerState, Mode
-from plenum.common.txn import REPLY, POOL_LEDGER_TXNS, \
-    LEDGER_STATUS, CONSISTENCY_PROOF, CATCHUP_REP, REQACK, REQNACK
-from plenum.common.types import Reply, OP_FIELD_NAME, f, HA, \
+from plenum.common.constants import REPLY, POOL_LEDGER_TXNS, \
```

Comment:
I'm not sure it's a good idea to have a module to collect all constants. I have sometimes done that in my career--only to regret it later when I realize that I have to #include or import my shared constants file everywhere. Should we be splitting the constants into more granular subsets? (This may not be a big deal if the number of constants is very small, and if they all need approximately the same visibility.)